librz/core/cmd/cmd_search_rop.c

```diff
@@ -727,3 +730,389 @@ static void rop_classify(RzCore *core, Sdb *db, RzList /*<char *>*/ *ropList, co
 
 	free(str);
 }
+
+RZ_API void rz_rop_constraint_free(void *data) {
```

Comment:
What is not related to the command handling, I recommend moving to a separate file in `librz/core`. E.g. `librz/core/crop.c` or something like that, where the main logic could be moved to.